install.sh

```diff
@@ -196,12 +196,20 @@ if [ "$1" = "install" ] || [ "$1" = "config" ]; then
   if [ "$_distro" = "Debian" ]; then #Help Debian cert problem.
     sed -i -e 's#CONFIG_SYSTEM_TRUSTED_KEYS="debian/certs/test-signing-certs.pem"#CONFIG_SYSTEM_TRUSTED_KEYS=""#g' .config
   fi
-  yes '' | make oldconfig
+  yes '' | make ${llvm_opt} oldconfig
   msg2 "Done"
 
   # apply linux-tkg patching script
   _tkg_srcprep
 
+  # source cpuschedset since _cpusched isn't set
+  source "$srcdir"/cpuschedset
+
+  # Uppercase characters are not allowed in source package name for debian based distros
+  if [ "$_cpusched" = "MuQSS" ] && [ "$_distro" = "Debian" ] || [ "$_distro" = "Ubuntu" ]; then
```

Comment:
This is equivalent to:
```
if ( [ "$_cpusched" = "MuQSS" ] && [ "$_distro" = "Debian" ] ) || [ "$_distro" = "Ubuntu" ]; then
```
Which isn't what we want, is it?

I guess we want:
```
if [ "$_distro" = "Debian" ] || [ "$_distro" = "Ubuntu" ] && [ "$_cpusched" = "MuQSS" ]; then
```

Reply:
oh yeah true lol

Reply:
ok fixed :)